.github/workflows/validate.yml

```diff
@@ -14,24 +14,24 @@ jobs:
       - name: Check if container changed in this PR
         id: check-dockerfile-changed
         run: |
-          # the SHA is target branch thanks to pull_request_target trigger
-          # add prints for debugging purposes
-          git log -1 --oneline ${{ github.sha }}
-          git diff --name-only ${{ github.sha }}
-          git diff --name-only ${{ github.sha }} | grep -q dockerfile/anaconda-ci/ || exit 2
```

Comment:
Just a note this is not `HEAD` as you are writing in the commit message but master because of `pull_request_target`. So it should work as expected, I've tested that. However, I like your solution more so no problem with changing this.

Reply:
Ah right, `github.event.pull_request.head.sha != github.sha`. Then the previous code makes more sense indeed. Sorry for the mix-up.